Content.Server/GameTicking/GameTicker.RoundFlow.cs

```diff
@@ -501,6 +537,18 @@ private void UpdateRoundFlow(float frameTime)
             {
                 RoundLengthMetric.Inc(frameTime);
             }
+            else if (RunLevel == GameRunLevel.PostRound)
+            {
+                _expandPvsEntities.Clear();
```

Comment:
I know this only happens during the post round, but this should probably still not update every tick for the sake of performance. 

Players will often have at least 10 entities attached to them, so on a 80 player server this will still iterate ~1k entities every tick, And if they all have backpacks full of stuff, it will be much higher.

Reply:
After looking at it again, its probably not that bad and would only make PVS/game-state slightly slower. #19083 basically goes back to just iterating all children every tick,